### DISCUSSION TITLE
The query parameter changesNotSentForReview must not be set.

### DISCUSSION THREAD

**Discussion by miguelmobilat** [codemagic-ci-cd/codemagic-docs#2699] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

== Gathering artifacts ==

== Publishing artifacts ==

Publishing artifact app-production-release.aab
Publishing artifact mapping.txt

Publishing app-production-release.aab to Google Play
Published app-production-release.aab to track internal
Google Play failed to upload artefacts. Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.: {
    "error": {
        "code": 400,
        "message": "Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.",
        "status": "INVALID_ARGUMENT"
    }
}

Build failed :|

Publishing failed :|
Google Play failed to upload artefacts. Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.: {
    "error": {
        "code": 400,
        "message": "Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.",
        "status": "INVALID_ARGUMENT"
    }
}
help please!

<img width="1018" alt="Screenshot 2024-08-29 at 10 13 58 AM" src="https://github.com/user-attachments/assets/b26cbfb2-c7dc-414c-8fea-c56f925d7655">

### Expected results

upload google play success

### Actual results

error== Gathering artifacts ==

== Publishing artifacts ==

Publishing artifact app-production-release.aab
Publishing artifact mapping.txt

Publishing app-production-release.aab to Google Play
Published app-production-release.aab to track internal
Google Play failed to upload artefacts. Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.: {
    "error": {
        "code": 400,
        "message": "Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.",
        "status": "INVALID_ARGUMENT"
    }
}

Build failed :|

Publishing failed :|
Google Play failed to upload artefacts. Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.: {
    "error": {
        "code": 400,
        "message": "Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.",
        "status": "INVALID_ARGUMENT"
    }
}

**Comment by himesh-cm** [comment]
Hey @miguelmobilat, This is a common issue. Please check our documentation for a detailed explanation and resolution. - https://docs.codemagic.io/troubleshooting/common-google-play-errors/#changesnotsentforreview-errors

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
